.github/workflows/check.yaml

```diff
@@ -3,6 +3,8 @@ on:
   push:
     branches:
       - main
+  schedule:
+    - cron: '39 4,15 * * *'
```

Comment:
I think the [cron encoding](https://docs.github.com/en/actions/reference/events-that-trigger-workflows#schedule) is sufficiently opaque that it's worth adding a comment above this line to explain what it's scheduling in a human-readable form, please.

Reply:
Good idea, thanks. e8b721e